microservices/agreement-process/prod/values.yaml

```diff
@@ -0,0 +1,93 @@
+name: "interop-be-agreement-process"
+techStack: "nodejs"
+
+serviceAccount:
+  roleArn: "arn:aws:iam::697818730278:role/interop-be-agreement-process-prod-es1"
+
+service:
+  create: true
+  containerPort: 8088
+
+configmap:
+  EVENTSTORE_DB_SCHEMA: "{{.Values.namespace}}_agreement"
+  S3_BUCKET: "interop-application-documents-{{.Values.namespace}}-es1"
+  ACCEPTED_AUDIENCES: "interop.pagopa.it/ui,interop.pagopa.it/m2m,interop.pagopa.it/internal"
+  AGREEMENT_CONTRACTS_PATH: "{{.Values.namespace}}/agreement/contracts"
+  CONSUMER_DOCUMENTS_PATH: "{{.Values.namespace}}/agreement/consumer-docs"
+
+deployment:
+  flywayInitContainer:
+    create: true
+    migrationsConfigmap: "event-store-migrations"
+    env:
+      FLYWAY_CREATE_SCHEMAS: "true"
+      FLYWAY_PLACEHOLDER_REPLACEMENT: "true"
+      FLYWAY_URL: jdbc:postgresql://$(POSTGRES_HOST):$(POSTGRES_PORT)/$(POSTGRES_DB)
+    envFromConfigmaps:
+      FLYWAY_SCHEMAS: "{{.Values.name}}.EVENTSTORE_DB_SCHEMA"
+      FLYWAY_PLACEHOLDERS_APPLICATIONSCHEMA: "{{.Values.name}}.EVENTSTORE_DB_SCHEMA"
+      POSTGRES_DB: "common-event-store.EVENTSTORE_DB_NAME"
+      POSTGRES_HOST: "common-event-store.EVENTSTORE_DB_HOST"
+      POSTGRES_PORT: "common-event-store.EVENTSTORE_DB_PORT"
+    envFromSecrets:
+      FLYWAY_USER: "event-store.POSTGRES_USR"
+      FLYWAY_PASSWORD: "event-store.POSTGRES_PSW"
+  env:
+    PORT: "8088"
+    HOST: "0.0.0.0"
+    EVENTSTORE_DB_USE_SSL: "true"
+    SERVICE_VERSION: "{{.Values.deployment.image.tag}}"
+    FEATURE_FLAG_APPLICATION_AUDIT_STRICT: "false"
```

Comment:
unless specific exceptions are required, feature flags should be in the common configmap.
Same for other occurrences